server/src/infra/repositories/filesystem.provider.ts

```diff
@@ -92,4 +92,8 @@ export class FilesystemProvider implements IStorageRepository {
       total: stats.blocks * stats.bsize,
     };
   }
+
+  async readdir(folder: string): Promise<string[]> {
+    return fs.readdir(folder);
+  }
```

Comment:
I think you can also do `readdir = readdir` if you do `import { readdir } from 'fs/promises'`